homeassistant/components/ubee/device_tracker.py

```diff
@@ -1,4 +1,9 @@
-"""Support for Ubee router."""
+"""
+Support for Ubee router.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/device_tracker.ubee/
```

Comment:
We don't include the url in the docstring anymore. Just keep the very first line.

Reply:
Done.